WordPress/src/main/res/values/strings.xml

```diff
@@ -844,6 +844,11 @@
     <string name="stats_bar_chart_content_description">Graph of site views.</string>
     <string name="stats_overview_content_description">%1$s %2$s for period: %3$s, change from previous period - %4$s</string>
     <string name="stats_graph_updated">Graph updated.</string>
+    <string name="stats_expand_content_description">Expand</string>
+    <string name="stats_item_expanded">Item expanded</string>
+    <string name="stats_item_collapsed">Item collapsed</string>
+    <string name="stats_posting_activity_graph">Graph of posting activity.</string>
+    <string name="stats_list_item_description">%1$s: %2$s, %3$s: %4$s</string>
```

Comment:
This will totally raise some questions from translators :)

Reply:
I know :-), any ideas about how to make it nicer? should I add a comment?

Reply:
Hmm. This is a tricky one. I think since they use translated labels in it, it should be ok? We can think of something better if we get asked about it.